src/Panels/HyperspaceJumpPanel.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Hyperdrive\Panels;
+
+use Hyperdrive\Contracts\PanelContract;
+use Hyperdrive\Galaxy\Geography\Planet;
+use Hyperdrive\Panels\Options\HyperspaceJumpOptions;
+use Hyperdrive\Player\Navigator\HyperspaceJump;
+use Symfony\Component\Config\Definition\Exception\Exception;
+
+class HyperspaceJumpPanel extends BasePanel implements PanelContract
+{
+    public function __construct(protected HyperspaceJump $hyperspaceJump)
+    {
+        parent::__construct();
+    }
+
+    /**
+     * @throws Exception
+     */
+    public function selectionSection(): void
+    {
+        $hyperspaceJumpOptions = new HyperspaceJumpOptions();
+        $result = $this->cli->radio("Select option", $hyperspaceJumpOptions())->prompt();
+
+        if ($result === "quit") {
```

Comment:
Może dobrze by było zapisać `quit` w jakiejś stałej, albo w ogóle stworzyć osobną klasę słownikową ze stałymi zawierającymi wszystkie możliwe typy odpowiedzi?

Reply:
Na pewno by się przydało, ułatwiłoby to wprowadzanie ewentualnych zmian na stałych.